### PR TITLE
Tidy up bootstrap script

### PR DESCRIPTION
**Pull request by caglar10ur**
Also decrease tmpfs size to 64MB from 128 and start copying kernel
module directory as bind mount is not working as expected after a
switch-root.

### CODE REVIEW

**Comment by hmahmood** [comment]
lgtm

File: isos/bootstrap/bootstrap [hunk 1]
```diff
@@ -43,7 +38,7 @@ if mount -t ext4 /dev/disk/by-label/containerfs ${MOUNTPOINT}; then
     # enable full system functionality in the container
     echo "Publishing modules within container"
     mkdir -p ${MOUNTPOINT}/lib/modules
-    mount --bind /lib/modules ${MOUNTPOINT}/lib/modules
+    cp -pr /lib/modules/* ${MOUNTPOINT}/lib/modules/
```
Comment (fdawg4l) [inline isos/bootstrap/bootstrap]: Once the modules are `insmod`'ed, do we need them accessible on the filesystem anymore?

Comment (caglar10ur) [inline isos/bootstrap/bootstrap]: We do if we want to provide more (like nfs/cifs mounting, enabling some other kernel feature etc)

**Comment by fdawg4l** [comment]
lgtm

**Comment by hickeng** [comment]
LGTM

**Comment by caglar10ur** [comment]
♻️

**Pull request merged**